RecoParticleFlow/PFTracking/src/PFTrackAlgoTools.cc

```diff
@@ -0,0 +1,38 @@
+#include "RecoParticleFlow/PFTracking/interface/PFTrackAlgoTools.h"
+namespace PFTrackAlgoTools {
+unsigned int getAlgoCategory(const reco::TrackBase::TrackAlgorithm& algo,bool hltIterativeTracking) {
+  switch (algo) {
+  case reco::TrackBase::ctf:
+  case reco::TrackBase::duplicateMerge:
+  case reco::TrackBase::initialStep:
+  case reco::TrackBase::lowPtTripletStep:
+  case reco::TrackBase::pixelPairStep:
+  case reco::TrackBase::jetCoreRegionalStep:
+    return 0;
```

Comment:
Well no.. here we group the iterations to some purity categories that are using to apply Dpt/pt cuts.. so if I did enum it would be cat0,cat1 etc...